nextflow_schema.json

```diff
@@ -0,0 +1,563 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "$id": "https://raw.githubusercontent.com/nf-core/mhcquant/master/nextflow_schema.json",
+    "title": "nf-core/mhcquant pipeline parameters",
+    "description": "Identify and quantify peptides from mass spectrometry raw data",
+    "type": "object",
+    "definitions": {
+        "input_output_options": {
+            "title": "Input/output options",
+            "type": "object",
+            "fa_icon": "fas fa-terminal",
+            "description": "Define where the pipeline should find input data and save output data.",
+            "required": [
+                "input"
+            ],
+            "properties": {
+                "input": {
+                    "type": "string",
+                    "fa_icon": "fab fa-think-peaks",
+                    "description": "Input raw / mzML files",
```

Comment:
Maybe mention in the `description` as well that it should be a TSV file

Reply:
ok should be resolved